www/front_src/src/Resources/Details/tabs/Details/DetailsCard/DetailsChips.tsx

```diff
@@ -1,18 +1,17 @@
-import { equals } from 'ramda';
 import { useTranslation } from 'react-i18next';
 
 import { Grid, Typography } from '@mui/material';
 import makeStyles from '@mui/styles/makeStyles';
 
-import { labelGroups } from '../../../../translatedLabels';
 import { CriteriaNames } from '../../../../Filter/Criterias/models';
-import { ResourceDetails } from '../../../models';
-import { ResourceType } from '../../../../models';
+import { Category, Group } from '../../../models';
 
-import GroupChip from './GroupChip';
+import DetailsChip from './DetailsChip';
 
 interface Props {
-  details: ResourceDetails | undefined;
+  getType: () => CriteriaNames;
+  metaResources?: Array<Category | Group>;
```

Comment:
I don't think metaResource is the proper name for the entities as it can be confused with meta services

Reply:
> I don't think metaResource is the proper name for the entities as it can be confused with meta services

I think we can rename it to `chips`